build_tools/bazel_to_cmake/bazel_to_cmake_converter.py

```diff
@@ -75,6 +75,7 @@ def _convert_target_block(name, target):
   #   //iree/tools:iree-translate
   #   iree::tools::iree-translate
   #   iree_tools_iree-translate
+  # TODO(scotttodd): update this example, or remove the code if no longer used?
```

Comment:
Seems like you can easily update the example right now?

Reply:
Well I'm not sure if this is still used and if we'd want to keep it around if not. This specific example is now just `//tools:iree-opt` -> `iree-opt`, using the regex replacement. I don't see any other binaries using aliases that go down this full path. There _are_ still other binaries like this, using the within-the-same-package shorthand: https://github.com/google/iree/blob/f3e9839ee5326ed984f418c1cd7662b21ed91131/runtime/src/iree/runtime/demo/CMakeLists.txt#L19-L36

Reply:
Reworked that comment a bit, including removing the example.